README.md

```diff
@@ -1845,6 +1845,7 @@ A [low-code](https://en.wikipedia.org/wiki/Low-code_development_platform) develo
 
 - [Appsmith](https://www.appsmith.com/) - Cloud or self-hosted open-source platform to build admin panels, CRUD apps and workflows. Build everything you need, 10x faster. ([Source Code](https://github.com/appsmithorg/appsmith)) `Apache-2.0` `Java/Docker`
 - [Budibase](https://www.budibase.com) - Build and automate internal tools, admin panels, dashboards, CRUD apps, and more, in minutes. Budibase is the open source alternative to Outsystems, Retool, Mendix, Appian. ([Source Code](https://github.com/Budibase/budibase)) `GPL-3.0` `Nodejs`
+- [DronaHQ](https://www.dronahq.com/) — DronaHQ - a low code platform helps engineering teams and product managers to build internal tools, custom user journeys, digital experiences, automations, custom admin panels, operational apps 10X faster. 
```

Comment:
This entry is missing language, license and a link to the source code. Looking at the website it actually seems to be a hosted service and therefore is not suitable for this list.

Reply:
Ah, found some self hosting instructions at https://www.dronahq.com/self-hosted-signup/

Reply:
Since this appears to be closed source the entry would rather fit in https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/non-free.md

Reply:
Yes, so we give an option of self-hosting as well, since you mentioned the non-free list, I would like to let you know that we have 4 plans for users, in which the developer plan is forever free.

Reply:
Would it be possible if we can add to this list only under the "Software development - low code" category?

Reply:
> I would like to let you know that we have 4 plans for users, in which the developer plan is forever free.

That non-free is not about money, but about freedom. As it is explained at the top of the non-free readme:

> Software listed here does not meet the [Free Software](https://en.wikipedia.org/wiki/Free_software) definition. It implements their own licensing with restrictions and grants which you must check on each case. Restrictions may include limits on allowed use of the software, access to the source code, modification and further redistribution. This software can therefore contain anti user-freedom features, such as but not limited to: backdoors, user lock-in, sending personal data to a third party.

Reply:
This is correct, it's not Free/Libre and Open-Source Software, it must be moved to non-free.md. You can create a new section `Software Development - Low Code` there, the guideline _Any category you are creating has the minimum requirement of 3 items_ was never really enforced in non-free.md. Please also change the link to https://www.dronahq.com/self-hosted/.